hack/testing/prep-host

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# prep the environment to allow bailing early on failures
+# setup ansible
+WORKDIR=$(mktemp -d)
+ANSIBLE_RPM=${ANSIBLE_RPM:-ansible-2.2.1.0-1.el7.noarch}
```

Comment:
I thought there is a bug currently with that version of Ansible... shouldnt we be using 2.2.0.0 ?